appium/webdriver/common/multi_action.py

```diff
@@ -47,8 +47,12 @@ def add(self, *touch_actions):
             if self._touch_actions is None:
                 self._touch_actions = []
 
-            # deep copy, so that once they are in here, the user can't muck about
-            self._touch_actions.append(copy.deepcopy(touch_action))
+            try:
+                # deep copy, so that once they are in here, the user can't muck about
+                self._touch_actions.append(copy.deepcopy(touch_action))
+            except TypeError:
+                # For python3
+                self._touch_actions.append(copy.copy(touch_action))
```

Comment:
is deepcopy not present in py3?

Reply:
No, py3 has deepcopy, but deepcopy doesn't work for own class(e.g. TouchAction) in py3.

https://stackoverflow.com/questions/32593035/why-does-deepcopy-fail-when-copying-a-complex-object

In that case, ```__deepcopy__``` has to be implemented by ourselves.

e.g. at ```TouchAction```
```
    def __deepcopy__(self, memodict={}):
        new_obj = TouchAction(self._driver)
        for action in self._actions:
            new_obj._add_action(action['action'], action['options'])
        return new_obj
```

At least, I have to consider current impl again since copy is not equal to deepcopy.
Steps are as below.

1. Check how current ```deepcopy``` works. If no problem, I'd like to remove it.
1. Implement own ```__deepcopy__```

Reply:
we just have to decide if shallow copy is enough for such case. In case it is not enough then I would rather suggest implementing __deepcopy__

Reply:
Sorry for my missing check, but here ```deepcop``` doesn't work also in python2.
I've checked python2 behavior after adding try-catch.
With my previous codes, shallow copy worked in both py2 and py3.

### Summary
* Here ```deepcopy``` doesn't work both in py2 and py3..
   * py2: TBU
   * py3: https://stackoverflow.com/questions/32593035/why-does-deepcopy-fail-when-copying-a-complex-object
* Replaced ```deepcopy``` with ```copy```
   * Verified ```test_driver_multi_tap``` working
   * Verified ```test_driver_multi_tap``` working with objects with same value as below. Objects with same values are recognized as different one.

```
positions = [(100, 200), (100, 400), (100, 200)]
self.driver.tap(positions)
```

log by ```print(actions)``` at [json_wire_gestures@muti_action.py](https://github.com/appium/python-client/blame/129bed8f6f4e4c5e2145fde5f53a89fcd4845654/appium/webdriver/common/multi_action.py#L77)
```
[[{'action': 'press', 'options': {'x': 100, 'y': 200}}, {'action': 'release', 'options': {}}], [{'action': 'press', 'options': {'x': 100, 'y': 400}}, {'action': 'release', 'options': {}}], [{'action': 'press', 'options': {'x': 100, 'y': 200}}, {'action': 'release', 'options': {}}]]
```


Any comments to my checks are welcome.